test/simulation/docker-build.sh

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
-docker-compose build $@
-
+if [[ $@ == "connext" || $# == 0 ]]
+then
+    mkdir -p temp
+    pushd temp
+    git clone https://github.com/ConnextProject/indra.git
+    cd indra
+    git checkout ab4daf9b63726ab5ca3d1903d910a2682b8855ef
```

Comment:
think we can upgrade to `7.0.0.alpha6` here since our simnet is now also running that